src/themes/masschallenge/elements/container.overrides

```diff
@@ -0,0 +1,24 @@
+/*******************************
+         Theme Overrides
+*******************************/
+
+@media screen and (min-width: 768px) and (max-width: 1440px) {
+.container{
+    display: grid;
+    grid-column: 2;
+    grid-template-columns: 220px 960px;
+    grid-template-areas: "topbar topbar" "sidebar content";
+    grid-template-rows: 55px calc(100vh - 55px - 2em);
+    grid-gap: 2em;
+    }
+}
+
+@media screen and (min-width: 315px) and (max-width: 425px) {
```

Comment:
@shakirandagire is there a reason why we didn't use Semantic breakpoint variables e.g. @computerBreakpoint, @tabletBreakpoint, etc instead of hardcoding the values?

Reply:
I have made the changes as requested.

Reply:
Also, why specify a min size here? What happens if the screen is smaller? Hard to do, but it would break. It seems there is a gap in layout specification between 425px and 768px. I think we only need two layouts:

- Tablet portrait and below (768px and lower) display the mobile tab bar
- Everything else (ie any larger screens) see the sidebar navigation

Reply:
I have worked on the screen sizings